docs/pages/installation.mdx

```diff
@@ -48,55 +48,391 @@ supports most features on Windows 10 and later.*
-|--------------------------------------------------------------------------------|---------------|--------------------------|----------|-------------|------------------------|----------------|
-| [Enhanced Session Recording](./server-access/guides/bpf-session-recording.mdx) | Kernel  v5.8+ | 11, or 10 with backports | 20.04.2+ | 9+          | 2 (post 11/2021), 2023 | 12 SP5, 15 SP5 |
-| Automatic Updates                                                              | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
-| Installation through apt/yum/zypper repos                                      | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
+Valid editions are:
 
-<Admonition type="note">
-`apt`, `yum`, and `zypper` repos don't expose packages for all distribution variants.
-When following installation instructions, you might need to replace `ID` with
-`ID_LIKE` to install packages of the closest supported distribution.
+| Value        | Edition                           |
+|--------------|-----------------------------------|
+| `cloud`      | Teleport Enterprise Cloud         |
+| `enterprise` | Teleport Enterprise (Self-Hosted) |
+| `oss`        | Teleport Community Edition        |
+| `team`       | Teleport Team                     |
 
-Currently supported distributions (and `ID`) are:
-- RHEL >= 7 (`rhel`)
-- CentOS >= 7 (`centos`)
-- Debian >= 9 (`debian`)
-- Ubuntu >= 16.04 (`ubuntu`)
-- Amazon Linux 2 and 2023 (`amzn`)
-- SLES >= 12 SP5, >= 15 SP5 (`sles`)
+If running Teleport Enterprise Cloud, Teleport agents must be running version
+(=cloud.version=) or one major version below.
 
-</Admonition>
+### Using Teleport packages and archives
 
+Read the instructions in this section for how to install Teleport on a Linux
+server with your package manager of choice. Note that the [one-line installation
+script](#one-line-installation-script) runs these commands for you after
+determining which package manager to use.
 
-### Installation instructions
+#### apt
 
-(!docs/pages/includes/permission-warning.mdx!)
+Run the following commands to install Teleport binaries on Linux systems that
+use the `apt` package manager. Replace `TELEPORT_PKG` with `teleport-ent` for
+Teleport Enterprise Cloud, Teleport Team, and Teleport Enterprise (Self-Hosted):
 
-(!docs/pages/includes/install-linux.mdx!)
+```code
+$ TELEPORT_PKG=teleport
+# Download the Teleport PGP public key
+$ sudo curl https://apt.releases.teleport.dev/gpg \
+-o /usr/share/keyrings/teleport-archive-keyring.asc
+# Source variables about OS version
+$ source /etc/os-release
+# Add the Teleport APT repository for v(=teleport.major_version=). You'll need to update this
+# file for each major release of Teleport.
+$ echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] \
```

Comment:
These instructions are incorrect for Cloud. Cloud users should be using stable/cloud channel, and must also install teleport-ent-updater to be enrolled in automatic upgrades.